dali/pipeline/data/tensor_list_test.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2017-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Looks like that update is not needed after all?

Reply:
Done, but let me complain that the script that I use make it hard.